Update docs with new features (#38)
Rebilly/framepay-docs: docs/configuration/properties.md

<<< @/docs/.vuepress/public/examples/example-publishablekey-1/index.js

## organizationId

> Your organization identifier in scope of which need to perform request (if not specified, the default organization will be used).

Reviewer: ```suggestion
Your organization identifier, in the scope of which requests are performed. (If it is not specified, the default organization will be used).
```
